### DISCUSSION TITLE
Next release (v0.11.0)?

### DISCUSSION THREAD

**Discussion by kasperah** [elixir-protobuf/protobuf#299] [Q&A] [answered]
I have some issues with Dialyzer breaking when using version protobuf 0.10.0, which seems to be fixed in the latest master (48c39d4fd5eb80fcb343d2cdf2dce6d1433e716d)

Do you have any idea when the next release will be ready?

best regards

**Comment by whatyouhide** [comment, marked as answer]
I'll try to get it released this or next week.

**Reply by kasperah** [reply]
sounds great! Thank you for the quick response and a great lib
